tests/_bases.py

```diff
@@ -13,10 +15,29 @@
 from tests import _helpers
 
 
```

Comment:
The diff (especially in unified view) is a little confusing just below here. I am not actually renaming `TestEmbedBase`.